core/ecschema-metadata/test/Metadata/Schema.test.ts

```diff
@@ -116,24 +109,24 @@ describe("Schema", () => {
       const testSchema = new Schema(new SchemaContext(), "TestSchema", "ts", 1, 1, 1);
       await (testSchema as MutableSchema).createEntityClass("TestEntity");
 
-      expect(await testSchema.getItem("TestEntity")).instanceof(ECClass);
-      expect(await testSchema.getItem<EntityClass>("TestEntity")).instanceof(EntityClass);
+      expect(ECClass.isECClass(await testSchema.getItem("TestEntity"))).to.equal(true);
+      expect(((await testSchema.getItem<EntityClass>("TestEntity")) as EntityClass).schemaItemType).to.equal(SchemaItemType.EntityClass);

```

Comment:
I don't think the as casts are necessary in this case and the ones below you can just check the schema item type